res/controllers/Reloop-Ready-scripts.js

```diff
@@ -0,0 +1,1146 @@
+
+// eslint-disable-next-line no-var, no-unused-vars
+var ReloopReadyInstance = {
+    init: function() {
+        ReloopReady.init();
+
+    }
+};
+
+class ReloopReady {
+    static init() {
+        ReloopReadyInstance = new ReloopReady();
+    }
```

Comment:
This looks rather complicated. Could you check if the code of the Z1 mapping would work here as well:
https://github.com/mixxxdj/mixxx/blob/23e91ac0bf7c86c5b03dc2d7048396be6de2ad5c/res/controllers/Traktor-Kontrol-Z1-scripts.js#L266

Reply:
That's what I would've preffered too, but it didn't work out due to naming issues when I tried. I'll try again soon though.